isis/tests/TestCsmModel.cpp

```diff
@@ -224,7 +224,7 @@ std::string TestCsmModel::constructStateFromIsd(const csm::Isd isd){
 
   json parsedIsd;
   isdFile >> parsedIsd;
-  // Only extract the first 2 parameters from the file
+  // Only extract the first 3 parameters from the file
```

Comment:
This model only has 2 parameters. I would just change this to say the we only extract the parameters used by the model.

Reply:
Thanks for catching this. I added one and then removed it.